doc/b-examples/i-inverse-kinematics.md

```diff
@@ -1,7 +1,84 @@
 # Inverse kinematics (clik)
 
+This example shows how to position the end effector of a manipulator robot to a given position.
+The example employs a simple Jacobian-based iterative algorithm, which is called closed-loop inverse kinematics (CLIK).
+
 ## Python
-\include i-inverse-kinematics.py
+\includelineno i-inverse-kinematics.py
+
+### Explanation of the code
+First of all, we import the necessary libraries and we create the `Model` and `Data` objects:
+\until data
+
+The end effector corresponds to the 6th joint
+\skipline JOINT
+
+and its desired position is given as
+\skipline xdes
+
+Next, we define an initial configuration
+\skipline q
+
+This is the starting point of the algorithm. *A priori*, any valid configuration would do.
+We decided to use the value returned by `neutralConfiguration`.
+For a simple manipulator such as the one in this case, it simply corresponds to an all-zero vector,
+but using this method generalizes well to more complex kinds of robots, ensuring validity.
+
+Next, we set some computation-related values
+\until DT
+corresponding to the desired position precision (a tenth of a millimeter will do), a maximum number of iterations (to avoid infinite looping in case the position is not reachable) and a positive "time step" defining the convergence rate.
+
+Then, we begin the iterative process.
+At each iteration, we begin by computing the end-effector pose for the current configuration value:
+\skip forwardKinematics
+\until R
+
+Next, we compute the error between the desired position and the current one. Notice we chose to express it in the local joint frame:
+\skipline err
+
+If the error norm is below the previously-defined threshold, we have found the solution and we break out of the loop
+\until break
+
+Otherwise, we search for another configuration trying to reduce the error.
+
+We start by computing the Jacobian, also in the local joint frame. Since we are only interested in the position, and not in the orientation, we select the first three lines
```

Comment:
we select only the first three lines corresponding to the translation part

Reply:
Ok